digitalidentity/share_session_builder.go

```diff
@@ -0,0 +1,82 @@
+package digitalidentity
+
+import (
+	"encoding/json"
+)
+
+// ScenarioBuilder builds a dynamic scenario
+type ShareSessionBuilder struct {
+	shareSession ShareSession
+	err          error
```

Comment:
where's this used?

Reply:
We will add examples but after next issues are done. Not yet used and will be addded before release.

Reply:
ok, but i recommend just not adding stuff that may/ will be needed for the future or aren't used, better to add them later when it's actually used so you can review the full changes together

Reply:
Sure but they will be used in tests and after all methods are added we will have examples in next prs.